tests/mocha/test_helpers/procedures.js

```diff
@@ -3,7 +3,7 @@
  * Copyright 2020 Google LLC
  * SPDX-License-Identifier: Apache-2.0
  */
-goog.module('Blockly.test.procedureHelpers');
+goog.declareModuleId('Blockly.test.helpers.procedures');
```

Comment:
The tests can't be converted to ES Modules until the test harness is modified so they are not loaded directly from `file://` URLs.

Reply:
(The result of which is that the Mocha tests "pass" because none of them even get loaded, so there can be no failures.  Zero failures === success, according to the test harness.)